cmake/FindOpenVX.cmake

```diff
@@ -0,0 +1,11 @@
+ocv_clear_vars(HAVE_OPENVX OPENVX_LIBS)
+
+set(OPENVX_INSTALL_DIR "" CACHE PATH "OpenVX install directory")
+
+if(OPENVX_INSTALL_DIR)
+  message(STATUS "Using OpenVX from ${OPENVX_INSTALL_DIR}")
+  ocv_include_directories(${OPENVX_INSTALL_DIR}/include)
+  link_directories(${OPENVX_INSTALL_DIR}/lib)
+  set(OPENVX_LIBS openvx openvx-c_model)
```

Comment:
I would make `OPENVX_LIBS` a `CACHE` variable, so user can overwrite it. OpenVX library might have another name, rather than `openvx`.